tests/step_implementation.js

```diff
@@ -100,12 +104,15 @@ step('Scroll', async() => {
     await scrollTo($('#myDIV'));
 
     // Scrolling the page
+    // All scroll methods need a default value.
+    // Please use the down arrow and set the right default value
     await scrollRight(200);
     await scrollLeft(100);
     await scrollDown(200);
     await scrollUp(100);
 
     // Scrolling a specific element
+    // Should work without passing scroll px value
```

Comment:
The default value for px is 100.

Reply:
Doesn't work, because there's no null check for `selector`